backend/build.gradle

```diff
@@ -93,7 +93,6 @@ subprojects {
     test {
         jvmArgs("--add-opens=java.base/java.util=ALL-UNNAMED", "--add-opens=java.base/java.lang=ALL-UNNAMED")
         useJUnitPlatform()
-        failFast true
```

Comment:
need for this PR ? , if so why

Reply:
then you always needs to wait for all the tests to run and fail ( failing due to time out may take longer), suggest to set to false when running in CI. This is also related to the issue of trying to only use integration tests, as the system grows the running tests becomes slower